src/main/java/games/strategy/engine/chat/ChatPanel.java

```diff
@@ -26,7 +26,7 @@
  */
 public class ChatPanel extends JPanel implements IChatPanel {
   private static final long serialVersionUID = -6177517517279779486L;
-  static int s_divider_size = 5;
+  private static final int DIVIDER_SIZE = 5;
```

Comment:
This field is not modified after it is initialized.  It also is not used outside of this class.